big_tests/tests/service_domain_db_SUITE.erl

```diff
@@ -11,14 +11,13 @@ suite() ->
 
 all() ->
     [
-     core_lookup_works,
-     core_lookup_not_found,
-     core_static_domain,
-     core_cannot_insert_static,
-     core_cannot_disable_static,
-     core_cannot_enable_static,
-     core_get_all_static,
-     core_get_domains_by_host_type,
+     api_lookup_works,
+     api_lookup_not_found,
+     api_cannot_insert_static,
+     api_cannot_disable_static,
+     api_cannot_enable_static,
+     api_get_all_static,
```

Comment:
why to rename it?
get_all_static is function from the core module, not the api module.

Reply:
```
get_all_static(Node) ->
    rpc(Node, mongoose_domain_api, get_all_static, []).
```